ppr-ui/src/utils/date-helper.ts

```diff
@@ -18,11 +18,12 @@ export function format12HourTime (date: Date): string {
 
 /** Converts date to display format. */
 export function convertDate (date: Date, includeTime: boolean, includeTz: boolean): string {
-  if (!includeTime) return moment(date).format('MMMM D, Y')
+  // For just dates use the UTC timestamp otherwise the day of month is incorrect (1 day earlier).
+  if (!includeTime) return moment(date).utc().format('MMMM D, Y')
```

Comment:
Hey Doug, do you mean the date was getting converted incorrectly to pacific time or it was actually converting the utc date directly before? Just verifying: will this return the date in UTC or Pacific haha? We should always be showing pacific time in the UI by default I think

Reply:
Moment incorrectly formats the day of month without this change. 2021-08-30 was formatted as August 29, 2021.

Reply:
Right, but is that because it is 2021-08-30T00:00... in the db in UTC? Wouldn't that mean it is actually correct to show one day behind in pacific time?

Reply:
Right, thanks. I have restored convertDate and added time to the format call to force the correct local day.

Reply:
awesome 👍